ggshield/sca/filter.py

```diff
@@ -0,0 +1,60 @@
+import re
+from pathlib import Path
+from typing import List, Set
+
+from ggshield.scan import Scannable
+from ggshield.scan.file import get_files_from_paths
+
+
+# List of filepaths to ignore for SCA scans
+
+SCA_IGNORE_LIST = (
```

Comment:
Shouldn't this list be fetched from an API and cached locally?

Reply:
I don't think so. This was part of the other idea to filter SCA files, however, this would imply an other API and a third API call to perform an SCA scan. I think it's too much. It's just a list of filepaths, it's easy to maintain, and missing some elements in this list in a given version of GGShield will just imply that the first API call is larger.

Reply:
> it's easy to maintain

I agree this is probably ok for this usecase, but any change is not so easy to deploy, since we don't control the version of ggshield used by our clients.

Reply:
Indeed, but I don't think missing some elements here for some users is sufficiently critical to maintain an API. However, from a user point of view, they should be able to add other directories here: this is possible with the `exclude_regex` parameter.